src/common/containers/ProjectList/index.jsx

```diff
@@ -62,6 +63,7 @@ class ProjectListContainer extends Component {
           <Link to={projectURL}>{project.name}</Link>
         ) : project.name,
         state: cycle.state,
+        week: cycle.startOfWeek.mmddyy,
```

Comment:
With this value (renamed to `weekStartedAt`) in the graphql schema simplified (now just a date), we'll need to do the formatting that we need on the client side. The value assigned to the `week` prop in the table model can be the raw date wrapped in a formatting util function.

Example:
```js
import {formatDate} from 'src/common/util/format'
.
.
.
const ProjectModel = {
  ...
  week: formatDate(cycle.weekStartedAt),
  ...
}
```